static/gsApp/views/seerAutomation/seerAutomationProjectList.tsx

```diff
@@ -76,17 +86,89 @@ export function SeerAutomationProjectList() {
     setPage(p => p + 1);
   };
 
+  const allSelected = selected.size === projects.length && projects.length > 0;
+  const toggleSelectAll = () => {
+    if (allSelected) {
+      // Unselect all projects
+      setSelected(new Set());
+    } else {
+      // Select all projects
+      setSelected(new Set(projects.map(project => project.id)));
+    }
+  };
+
+  const toggleProject = (projectId: string) => {
+    setSelected(prev => {
+      const newSet = new Set(prev);
+      if (newSet.has(projectId)) {
+        newSet.delete(projectId);
+      } else {
+        newSet.add(projectId);
+      }
+      return newSet;
+    });
+  };
+
+  async function updateProjectsSeerValue(value: string) {
+    addLoadingMessage('Updating projects...', {duration: 30000});
+    try {
+      await Promise.all(
+        Array.from(selected).map(projectId => {
+          const project = projects.find(p => p.id === projectId);
+          if (!project) return Promise.resolve();
+          return new Promise((resolve, reject) => {
+            api.request(`/projects/${organization.slug}/${project.slug}/`, {
+              method: 'PUT',
+              data: {autofixAutomationTuning: value},
+              success: () => resolve(undefined),
+              error: () => reject(new Error(`Failed to update ${project.slug}`)),
+            });
+          });
```

Comment:
requestPromise exists and our RequestError has whatever you're looking for
```suggestion
          return api.requestPromise(`/projects/${organization.slug}/${project.slug}/`, {
              method: 'PUT',
              data: {autofixAutomationTuning: value},
            });
```